docs/Projects/TAD/index.md

```diff
@@ -0,0 +1,26 @@
+# TAD
+
+TAD is the acronym for **T**ransparency of **A**lgorithmic **D**ecision making, the goal is to make algorithmic systems more
+transparent by generating standardized reports on the algorithmic system for both it's technical element as well as the
+surrounding system. For both the model and the system the lifecycle is important and this needs to be taken into
+account.
```

Comment:
The last sentence of this paragraph does not really naturally follow from the previous sentences. Also it feels unfinished.